cmd/eval.go

```diff
@@ -272,6 +289,24 @@ func eval(args []string, params evalCommandParams) (err error) {
 			}
 			fmt.Println(string(bs))
 		}
+	case evalProfileOutput:
+		table := generateTable()
```

Comment:
I think this logic is going to get moved, but I would probably factor it out into a separate function.